tests/integration/blob.test.js

```diff
@@ -0,0 +1,161 @@
+import { u8aToU8a, u8aToString, u8aToHex } from '@polkadot/util';
+import { randomAsHex } from '@polkadot/util-crypto';
+
+import { DockAPI } from '../../src/api';
+
+import { createNewDockDID, createKeyDetail, getHexIdentifierFromDID } from '../../src/utils/did';
+import { FullNodeEndpoint, TestKeyringOpts, TestAccountURI } from '../test-constants';
+import { getPublicKeyFromKeyringPair } from '../../src/utils/misc';
+import { verifyCredential } from '../../src/utils/vc';
+import { DockBlobIdByteSize } from '../../src/modules/blob';
+import Schema from '../../src/modules/schema';
+import exampleCredential from '../example-credential';
+import exampleSchema from '../example-schema';
+
+let account;
+let pair;
+let publicKey;
+let dockDID;
+let keyDetail;
+let txDid;
+let resultDid;
+let didDoc;
+let blobId;
+
+function errorInResult(result) {
+  try {
+    return result.events[0].event.data[0].toJSON().Module.error === 1;
+  } catch (e) {
+    return false;
+  }
+}
+
+describe('Blob Module', () => {
+  const dock = new DockAPI();
+
+  // Generate first key with this seed. The key type is Sr25519
+  const firstKeySeed = randomAsHex(32);
+
+  beforeAll(async (done) => {
+    await dock.init({
+      keyring: TestKeyringOpts,
+      address: FullNodeEndpoint,
+    });
+    done();
+  });
+
+  afterAll(async () => {
+    await dock.disconnect();
+  }, 10000);
+
+  beforeEach(async () => {
+    account = dock.keyring.addFromUri(TestAccountURI);
+    dock.setAccount(account);
+    pair = dock.keyring.addFromUri(firstKeySeed);
+    publicKey = getPublicKeyFromKeyringPair(pair);
+    dockDID = createNewDockDID();
+    keyDetail = createKeyDetail(publicKey, dockDID);
+    txDid = dock.did.new(dockDID, keyDetail);
+    resultDid = await dock.sendTransaction(txDid);
+    didDoc = await dock.did.getDocument(dockDID);
+    blobId = randomAsHex(DockBlobIdByteSize);
```

Comment:
You can create the DID once in beforeAll, this will make tests run faster as they will do 1 chain write less